src/main/java/com/onarandombox/multiverseinventories/command/prompts/GroupWorldsPrompt.java

```diff
@@ -51,8 +51,8 @@ public Prompt acceptInput(final ConversationContext conversationContext, final S
                 messager.normal(Message.GROUP_WORLDS_EMPTY, sender);
                 return this;
             }
-            group.getWorlds().clear();
-            group.getWorlds().addAll(worlds);
+            group.removeAllWorlds(false);
+            group.addWorlds(worlds, false);

```

Comment:
Why don't you update the config here?

Reply:
Bcu the should update the config after the shares are added when creating a new group.

Reply:
Ah okay, makes sense.